pages/1.11/administering-clusters/add-a-node/index.md

```diff
@@ -11,18 +11,18 @@ enterprise: false
 
 
 
-Agent nodes are designated as [public](/1.11/overview/concepts/#public-agent-node) or [private](/1.11/overview/concepts/#private-agent-node) during installation. By default, they are designated as private during [GUI][1] or [CLI][2] installation.
+Agent nodes are designated as [public](/1.11/overview/concepts/#public-agent-node) or [private](/1.11/overview/concepts/#private-agent-node) during installation. By default, they are designated as private during the GUI or CLI [installation](/1.11/installing/evaluation/cloud-installation/) method.
 
 ### Prerequisites:
 
-*   DC/OS is installed using the [custom](/1.11/installing/oss/custom/) installation method.
-*   The archived DC/OS installer file (`dcos-install.tar`) from your [installation](/1.11/installing/oss/custom/gui/#backup).
```

Comment:
/1.11/installing/#backup is incorrect. GUI and CLI installer pages are removed from 1.11 installing docs. Replace /1.11/installing/#backup with /1.11/installing/production/deploying-dcos/installation/

Reply:
I don't see /installing/#backup anywhere in this corrected text. Also, there is no #backup tag at all in the 1.11/installing/evaluation/cloud-installation/ page.

Reply:
There is no #backup on that page.